native/cocos/math/Quaternion.cpp

```diff
@@ -371,45 +371,75 @@ void Quaternion::toEuler(const Quaternion &q, bool outerZ, Vec3 *out) {
 }
 
 void Quaternion::fromMat3(const Mat3 &m, Quaternion *out) {
```

Comment:
It is better to unity the ts & cpp implementation.

Reply:
ok